src/coinjoin/client.cpp

```diff
@@ -134,7 +134,7 @@ void CCoinJoinClientQueueManager::ProcessDSQueue(const CNode& peer, CDataStream&
     }
 }
 
-void CCoinJoinClientManager::ProcessMessage(CNode& peer, CConnman& connman, std::string_view msg_type, CDataStream& vRecv)
+void CCoinJoinClientManager::ProcessMessage(const CTxMemPool& mempool, CNode& peer, CConnman& connman, std::string_view msg_type, CDataStream& vRecv)
```

Comment:
nit, in line with the argument order of all other `ProcessMessage`s, consider changing this to `CNode& peer, CConnman& connman, const CTxMemPool& mempool, std::string_view msg_type, CDataStream& vRecv`